app/controllers/User/StoreController.js

```diff
@@ -50,13 +50,12 @@ class StoreController {
     }
 
     async invoke(request, response) {
-        const user = await this.userRepository.create(request.body);
+        const userData = request.body;
```

Comment:
not sure if needed, you are using it only once
also you can do `const { body: userData } = request`